lib/pbench/test/unit/server/auth/test_auth.py

```diff
@@ -253,19 +253,25 @@ def test_create_w_roles(self):
 
 
 @pytest.fixture(scope="session")
-def rsa_keys() -> Dict[str, str]:
+def rsa_keys() -> Dict[str, Union[rsa.RSAPrivateKey, str]]:
```

Comment:
You didn't change this, but changing the line highlights it ... now that we're using Python 3.9 we can use `dict[...]` instead of `typing.Dict`. I've been trying to gradually fix type hints as I hit them.